### ISSUE TITLE
Rework the TEI-XML-view

### ISSUE THREAD

**Issue by martinantonmueller** [arthur-schnitzler/schnitzler-briefe-static#102]
The legacy-xslt that creates the TEI-view has  a flaw that it introduces whitespace for reading purposes, for example immediately after an `<p>`-Element, e.g. https://schnitzler-briefe.acdh.oeaw.ac.at/L03186.xml 
This of course makes it readable but not immediately usable as somebody who copies the code would have to remove the whitespace. Maybe there is a better solution? At least an additional link to https://github.com/arthur-schnitzler/schnitzler-briefe-data/blob/main/data/editions/L03186.xml would be better

Eye this, @csae8092 – as it applies to DSE-static-cookie-cutter in general

**Comment by csae8092** [comment]
Not sure if we discussed this IRL already, but opening https://schnitzler-briefe.acdh.oeaw.ac.at/L03186.xml in the browser makes the browser reformat the file (as you described). But if I than click in the browser "save file" ... (or how this might be called) the "original" file is downloaded and any intendation is kept from the original. 
So I'd suggest to add some quick explanation text to the download/tei icon

**Comment by martinantonmueller** [comment]
Thank you, @csae8092! Might be that we discussed it in IRL and I forgot. But wouldn't it make sense to just link to the github-XML? There one has download-buttons etc.

**Comment by csae8092** [comment]
linking to the github-xml is IMHO a bad idea, because this is only an html file, so I can't access the actual XML file which I think is the worst thing to do; 
you could link to https://raw.githubusercontent.com/arthur-schnitzler/schnitzler-briefe-data/main/data/editions/L03186.xml but there the server claims that you'll receive a text-file (and therfore it looks shitty) and I'm not sure if Github keeps those "raw." URLs stable

so, I'd keep it as it is; maybe add some helptext; or add another icon linking to https://github.com/arthur-schnitzler/schnitzler-briefe-data/blob/main/data/editions/L03186.xml

**Comment by martinantonmueller** [comment]
yes, the last link in your post would have been the one I'd use. But I will reflect on your points and then maybe make a decision. Or leave it as it is.
